docs/spec/request.json

```diff
@@ -58,7 +58,7 @@
             "properties": {
                 "raw": {
                     "type": ["string", "null"],
-                    "description": "The raw, unparsed URL of the request, e.g https://example.com:443/search?q=elasticsearch#top.",
+                    "description": "The raw, unparsed URL of the HTTP request line, e.g https://example.com:443/search?q=elasticsearch. This URL may be absolute or relative. For more details, see https://www.w3.org/Protocols/rfc2616/rfc2616-sec5.html#sec5.1.2.",
```

Comment:
Afair this was specifically called `request.url` and not `http.url`, so not sure if the description is proper. As this was heavly driven by the experience and need of the @elastic/apm-agent-devs , could you please have a look at this.

Reply:
The reason why we renamed this from `http.url` to `request.url` was not because it wasn't http related, but because we needed to also add information about the response. So having one object called `http` and another called `response` was confusing. So we renamed them to `request` and `response`.

But it might have been better if the property clearly stated that it was http related.

Reply:
FWIW, I'd love it if we had `{"http":{"request":..., "response":...}}`

Reply:
That would be much better. Though that would deviate from our "property name" => "tab headline" rule in the APM UI. But it might be worth it. We'll have to discuss this with that in mind though /cc @sqren

Reply:
If that structure makes more sense, the UI should be able to accommodate that.

Reply:
Just remember that this is a backwards incompatible change, and any user (not just the APM UI) would need to change their code to accommodate this. I'm not sure if it's worth the effort.

Also, I'm really not a huge fan of deeply nested data structures, this would add another layer to an already quite deeply nested `request` object

Reply:
Moved that part of the discussion to a separate issue: https://github.com/elastic/apm-server/issues/794